jsonschema/protocols.py

```diff
@@ -0,0 +1,152 @@
+"""
+typing.Protocol classes for jsonschema interfaces.
+"""
+
+# for reference material on Protocols, see
+#   https://www.python.org/dev/peps/pep-0544/
+
+from typing import Any, ClassVar, Iterator, Optional
+
+try:
+    from typing import Protocol, runtime_checkable
+except ImportError:
+    from typing_extensions import Protocol, runtime_checkable
+
+from ._format import FormatChecker
```

Comment:
I use absolute imports always essentially, so:

```suggestion
from jsonschema._format import FormatChecker
```

and below.